packages/application-to-register-webapp/src/views/layout.html

```diff
@@ -80,10 +80,9 @@ <h1 class="govuk-heading-xl">Default page template</h1>
 
 {% block bodyEnd %}
   {# Run JavaScript at end of the <body>, to avoid blocking the initial render. #}
-    <script src="{{ assetPath }}/jquery.js"></script>
-    <script src="{{ assetPath }}/all.js"></script>
-    <script src="{{ assetPath }}/moj/all.js"></script>
-    <script>window.GOVUKFrontend.initAll()</script>
-    <script>window.MOJFrontend.initAll()</script>
-    <script src="{{ assetPath }}/js/core.js"></script>
```

Comment:
@reneli5 @pwadmore-ea Hi Rene, just found an issue rebasing with this work, do you know why this line for core.js inclusion was removed in your pull request?